Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -1132,6 +1158,19 @@ module.exports = {
       .setFunctionName('getLinearVelocityLength')
       .setManipulatedType('number');
 
+	aut
+      .addExpression(
+        'LinearVelocityLength',
+        t('Linear velocity Length'),
```

Comment:
Linear velocity

(velocity is like speed and already convey the meaning of a length)